index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav class="nav">
+        <a href="">video</a>
+        <a href="">sidebar</a>
+    </nav>
+    
+    
+    
+    <iframe class="video" width="560" height="315" src="https://www.youtube.com/embed/Yg6UhhV_K1s" 
+            title="YouTube video player" 
+            frameborder="0" 
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" 
+            allowfullscreen controls>
+    </iframe>
```

Comment:
@KingKowa Please kindly you should write why you are required changes, and explain in detail why he should use semantic tag